component-samples/demo/owner/Dockerfile

```diff
@@ -10,6 +10,9 @@ RUN useradd -ms /bin/bash fdo
 WORKDIR /home/fdo/
 RUN mkdir -p /home/fdo/lib /home/fdo/serviceinfo
 
+# Uncomment the following line to enable onDie support for Owner
+# COPY --chown=owner ./ondie_cache ./ondie_cache/
```

Comment:
chown is performed in line 24 for all files, not required here.

Reply:
Done.